use/identifying_to_services.rst

```diff
@@ -181,19 +181,26 @@ Services plugin
 ---------------
 
 The Services plugin comes with Supybot and should be an easy way to
-identify your bot, but SASL and ``username:password`` as server password
-are recommended over it. Start by loading Services with::
+identify your bot, but SASL is recommended over it. Start by loading
+Services with::
 
     load Services
 
 and then tell it what NickServ and ChanServ are called::
 
-    config plugins.services.nickserv NickServ
-    config plugins.services.chanserv ChanServ
+    config network [<network>] plugins.services.nickserv NickServ
+    config network [<network>] plugins.services.chanserv ChanServ
 
-Remember to replace NickServ/ChanServ with their real names if they have a
-different name on any network. Note that they must have the same name on
-all networks, and you must have the same password on all networks.
+``[<network>]`` is only necessary if the message isn't sent in the network
+itself. Remember to replace NickServ/ChanServ with their real names if they
```

Comment:
I am imitating the usual Supybot style, for example I copied this wording from `help channel ban add`